tests/views_test.py

```diff
@@ -1,5 +1,100 @@
-from src_api.views import helloworld
+from numpy import ndarray, result_type
+from src_api.views import *
+import shutil
+import os
 
+def TestBasics():
+    """
+    Sets the Basic Folder/Files for Testing.
 
-def test_helloworld():
-    assert helloworld() == "Hello world !"
+    Returns:
+        tuple: filenames and folder.
+    """
+    folder_name = os.path.join(app.config["APP_FOLDER"], "tests", "testfiles")
+    file_name = "test_file.wav"
+    file_name_copy = "test_file_copy.wav"
+    return folder_name,file_name,file_name_copy
+
+
+def TestBasicsCopy(shutil):
```

Comment:
See best practice formatting for functions (only use capital letters when declaring classes)

Reply:
<img width="470" alt="image" src="https://user-images.githubusercontent.com/93911252/148610822-f18004b8-d471-44c2-8c63-07888bdb4c83.png">